src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -85,21 +115,25 @@ end
 end
 
 # TODO: DGMulti. Fix for curved meshes.
-# Version for non-symmetric fluxes and sparse operators.
-@inline function hadamard_sum!(du, sparsity_pattern::AbstractSparseMatrix{Bool}, A,
+# Version for sparse operators and non-symmetric fluxes
+@inline function hadamard_sum!(du, A::LinearAlgebra.Adjoint{<:Any, <:SparseMatrixCSC},
```

Comment:
See above

Reply:
Good point - fixed in a291133a7cac379b46ff1f04b1c0a182a738188a by switching to `AbstractSparseMatrixCSC`.